### ISSUE TITLE
Focus ring in Chrome

### ISSUE THREAD

**Issue by creativetags** [kleinejan/titatoggle#23]
When you click on the toggle you get a focus ring or outline when using Chrome (I'm using latest version on Mac). Safari and Firefox don't have this issue.

I can't seem to get rid of it using the usual :focus {outline: none;} or
outline-style:none;
box-shadow:none;
border-color:transparent;

**Comment by kleinejan** [comment]
The outline should be there for accessibility reasons. It gives user a visual clue what element is selected. I know most designers don't 'like' the outline, but that's just the right way to go if you care about a11y. 
Good you noticed the lack of the outline in Safari and Firefox :) I should be fixing this.

On line 74:
.checkbox-toggle input[type="checkbox"] :focus + *:before { outline:solid @input-border-focus 2px; }

Change that to `outline: 0;` and your outline is removed.
But again that would be bad practice.

**Comment by creativetags** [comment]
Ok for a11y you could use `box-shadow: 0 0 3pt 2pt #66afe9;` instead of outline so you keep the round edge.

**Comment by kleinejan** [comment]
fair point
